sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/implementation/IdMetadata.java

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.security.keyvault.certificates.implementation;
+
+/**
+ * Model class containing metadata about a certificate identifier.
+ * <p>
+ * Creation of this class splits the ID into the sub parts vault URL, name, anc version.
+ */
+public final class IdMetadata {
```

Comment:
I made this manually as I needed something generic to pass around the certificate ID metadata that wasn't just a bunch of miscellaneous strings. And since this is just an implementation detail that is why it's in `implementation` instead of `implementation.models`.